Student.java

```diff
@@ -0,0 +1,18 @@
+
+    public record Student(String name, int grade, int classNum) {
+        public String toString() {
+            return "student{" + "name='" + name + '\'' + ", grade=" + grade + ", classNum=" + classNum + '}';
+        }
+
+        public boolean isStudentHS() {
+            if (Student.this.grade >= 9) {
```

Comment:
inside the class you can just access the grade like this
```suggestion
            if (grade >= 9) {
```